flow/e2eshared/e2eshared_test.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/PeerDB-io/peerdb/flow/model/qvalue"
 )
 
+func TestReadMissingFileToBytes(t *testing.T) {
+	if _, err := ReadFileToBytes("missing"); err == nil {
+		t.Error("ReadFileToBytes expected to fail on missing files")
```

Comment:
is it possible to assert on the exact error? can dump the function into a playground and check

Reply:
changed to check for `os.ErrNotExists`